components/brave_rewards/resources/shared/lib/rewards_urls.ts

```diff
@@ -12,4 +12,5 @@ export const contactSupportURL = 'https://community.brave.com/'
 export const supportedWalletRegionsURL = 'https://support.brave.com/hc/en-us/articles/6539887971469'
 export const aboutBATURL = 'https://brave.com/rewards'
 export const rewardsChangesURL = 'https://brave.com/rewards-changes'
+export const rewardsTourURL = 'https://brave.com/rewards-tour'
```

Comment:
Looks like this isn't live yet - is this still a work in progress?

Reply:
yes, the online tour has not been implemented yet, but @Miyayes said it's ok while this functionality is only in Nightly and the web team would put in a placeholder or a redirect.